docs/user/openstack/install_upi.md

```diff
@@ -0,0 +1,861 @@
+# Installing OpenShift on OpenStack User-Provisioned Infrastructure
+
+The User-Provisioned Infrastructure (UPI) process installs OpenShift in stages, providing opportunities for modifications or integrating with existing infrastructure.
+
+It contrasts with the fully-automated Installer-Provisioned Infrastructure (IPI) which creates everything in one go.
+
+With UPI creating the cloud (OpenStack) resources (e.g. Nova servers, Neutron ports, security groups) is the responsibility of the person deploying OpenShift.
+
+The installer is still used to generate the ignition files and monitor the installation process.
+
+This provides a greater flexibility at the cost of a more explicit and interactive process.
+
+
+## Prerequisites
+
+The requirements for UPI are broadly similar to the [ones for OpenStack IPI][ipi-reqs]:
+
+[ipi-reqs]: ./openstack#openstack-requirements
+
+- OpenStack account with `clouds.yaml`
+  - This document will use a cloud called `openstack`
```

Comment:
Do you think it makes sense to give some visual representation of the two layers of this paragraph ("general case" vs. "this document") by using _italic_ when referring to "this document"?